kafka/protocol/types.py

```diff
@@ -8,16 +8,20 @@
 def _pack(f, value):
     try:
         return pack(f, value)
-    except error:
-        raise ValueError(error)
+    except error as e:
+        raise ValueError("Error encountered when attempting to convert value: "
+                        "{} to struct format: '{}', hit error: {}"
```

Comment:
Maybe `{!r}` just in case str is overridden.

Reply:
I forgot to do this earlier, changed in https://github.com/dpkp/kafka-python/pull/1373